azure/project.yml

```diff
@@ -4,4 +4,27 @@ variables:
   service_base_path: booking-and-referral
   product_display_name: Booking and Referral FHIR API
   product_description: This proxy function as a centralized router for bookings and referrals services
-  spec_file: booking-and-referral.json
\ No newline at end of file
+  spec_file: booking-and-referral.json
+  ratelimiting: |
+    {
+      "app": {
+        "quota": {
+          "limit": 300,
+          "interval": 1,
+          "timeunit": "minute",
+          "enabled": true
+        },
+        "spikeArrest": {
+          "enabled": false
+        }
+      },
+      "proxy": {
+        "quota": {
+          "enabled": false
+        },
+        "spikeArrest": {
+          "ratelimit": "300000pm",
```

Comment:
i think this makes sense for the identity service but no for this proxy .. should be way less...